### PR TITLE
Adds config file dependency and relevant gitignore

### PR DESCRIPTION
**Pull request by joshuarpearce**
Better practice than having redacted in the string, also means it keeps the commit list a bit cleaner. You will however then need to make this file locally before the bot will work again

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 *.pyc
+config.py
+.vscode
```
Comment (bobrippling) [inline .gitignore]: Looks good, although could you move `.vscode` to your user git ignore file (`~/.config/git/ignore`)? As it isn't specific to the project. Ta muchly

File: .gitignore [hunk 2]
```diff
@@ -1 +1,4 @@
...
+ps4-games.txt
```
Comment (bobrippling) [inline .gitignore]: I'd just whack a newline on the end here to match the other files

File: .gitignore [hunk 3]
```diff
@@ -1 +1,4 @@
 *.pyc
+config.py
```
Comment (bobrippling) [inline .gitignore:2]: Would you mind updating the readme to say to create a `config.py` and whizz in an example of what it should contain? Also, I've just thought - whack yourself in the [credits](https://github.com/bobrippling/lunchbot/blob/master/PS4Bot.py#L391). The emoji choice is all yours

**Review by bobrippling — CHANGES_REQUESTED** [review]

**Comment by joshuarpearce** [comment]
Changes made.

Might be worth looking at a linter to keep styling consistent.

**Review by bobrippling — APPROVED** [review]

**Comment by bobrippling** [comment]
Merged this bad boi - I just took out the odd merge commits ([here](https://github.com/bobrippling/lunchbot/pull/2/commits/28385dada2d795e41abfa34fd3177a12b6fa448d)) that skewed the branch history

**Pull request closed without merging**